.c8rc.json

```diff
@@ -1,5 +1,5 @@
 {
-    "reporter": ["lcovonly", "cobertura"],
+    "reporter": ["lcovonly", "cobertura", "v8", "codecov"],

```

Comment:
Checked, and adding another reporter here does not affect the performance of the `report` stage, so we can keep this.